### PR TITLE
Makefile, release-1.2: Pin to KubeVirt v1.4.0-rc.0 and release-1.4

### PR DESCRIPTION
**Pull request by lyarwood**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 SHELL=/bin/bash
 
-export KUBEVIRT_VERSION = main
-export KUBEVIRT_TAG = main
+export KUBEVIRT_VERSION = v1.4.0-rc.0
+export KUBEVIRT_TAG = release-1.4
```
Comment (lyarwood) [inline Makefile:4]: FWIW `_VERSION` and `_TAG` are bad names IMHO but I'll rework this in a separate PR on main.

**Review by 0xFelix — APPROVED** [review]
/approve
/lgtm

**Pull request merged**